verilog/tools/syntax/export_json_examples/print_modules.py

```diff
@@ -0,0 +1,133 @@
+#!/usr/bin/env python3
```

Comment:
There is no entry for this in the BUILD file; this should become a `py_binary()`
(same for `print_tree.py`)

Reply:
done https://github.com/google/verible/pull/616/commits/c6e84c519bba51dc6ffecb67379663307521c3c4